assets/src/design-system/components/checkbox/index.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import propTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { Checkmark } from '../../icons';
+
+const FlexContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const Border = styled.div(
+  ({ disabled }) => css`
+    ${!disabled &&
+    css`
+      margin: 2px;
+
+      :focus-within {
+        border: 2px solid #5d96eb;
+        border-radius: 6px;
+        margin: 0;
+      }
```

Comment:
I didn't think to tell you about this - I'm sorry! 

We have a helper called `focusableOutlineCSS` that will set up the focus within for you and use the polyfill. It's not updated in the designs, but we're ditching the padding between the focused "outline"(border) and the element as a whole because there's cases where it's extremely hard to maintain proper aspect and not worth the time it was adding.  

What you did with the border as a div is super clever, but i think we can probably get around nesting this div with the above. Again, this is my bad for not communicating this. I'm sorry!

Reply:
Chatted about this over slack but adding here for visibility:

The `input` has `opactity: 0` so using `focusableOutlineCSS` didn't show up. The borders are being applied on the outer div, but reducing the white space between the focus outline and the element let us remove two extra divs that we were using (`Borders` and `FlexboxContainer`)